kafka/signed_blinded_token_issuer_handler.go

```diff
@@ -0,0 +1,124 @@
+package kafka
+
+import (
+	"bytes"
+	"fmt"
+
+	crypto "github.com/brave-intl/challenge-bypass-ristretto-ffi"
+	avroSchema "github.com/brave-intl/challenge-bypass-server/avro/generated"
+	"github.com/brave-intl/challenge-bypass-server/btd"
+	cbpServer "github.com/brave-intl/challenge-bypass-server/server"
+	"github.com/rs/zerolog"
+)
+
+/*
+ BlindedTokenIssuerHandler emits signed, blinded tokens based on provided blinded tokens.
+ @TODO: It would be better for the Server implementation and the Kafka implementation of
```

Comment:
can we make an issue to track this follow up?

Reply:
https://github.com/brave-intl/challenge-bypass-server/issues/53